package.json

```diff
@@ -35,21 +20,107 @@
     "examples/shortcodes",
     "examples/terminal"
   ],
+  "scripts": {
+    "bootstrap": "yarn && lerna bootstrap && lerna link --force-local",
+    "build": "r -f cjs,esm -o dist --exclude 'babel-plugin-*,create-mdx,*loader,mdx,parcel-plugin-mdx,remark-*,runtime,*util'",
```

Comment:
Is this intended to be rollup or another command?

Reply:
It's a wrapper [@pkgr/rollup](https://github.com/rx-ts/pkgr/tree/master/packages/rollup) which is mush simple to use.

Reply:
gotcha, good to know, it confused me for a bit because some shells have [`r` as a command that repeats the previous command](https://linux.die.net/man/1/zshbuiltins)